report/src/introduction/agile_approach.tex

```diff
@@ -1,29 +1,27 @@
 \section{Implementing Agile Project Management with Scrum}
 Developing a software product for a business can be a complex process.
-Incremental agile approaches have become popular to address this complexity, as they allow for faster development and adaptation to changes.\cite{sommervilleSoftwareEngineering2016}
-In incremental agile approaches, a system is developed as a series of increments, each adding functionality to the system.
-Between these increments, one gathers feedback from users to prioritize functionality for the next increment.
+Incremental agile approaches have become popular to address this complexity, as it allows for fast adaptation to changes in user needs. \cite{sommervilleSoftwareEngineering2016}
+In incremental-agile approaches, system development is considered a series of increments, each adding functionality to the system.
```

Comment:
Why add the dash now and not elsewhere?
Don't think this change is better. The previous describes the approach better imo.

Reply:
Indeed. Undone.